mlx/backend/accelerate/softmax.cpp

```diff
@@ -331,7 +331,7 @@ void Softmax::eval_cpu(const std::vector<array>& inputs, array& out) {
       return x_copy;
     }
   };
-  array in = check_input(std::move(inputs[0]));
+  array in = check_input(inputs[0]);
```

Comment:
Nice, this was left because initially it was written like this

```
array in_pre = inputs[0]; // copy first
auto in = check_input(std::move(in_pre));
```